examples/api/autosuggest.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const express = require('express');
+const multer = require('multer');
+const upload = multer();
+
+// eslint-disable-next-line new-cap
+const examples = express.Router();
+const US_CAPITAL_CITIES = [
+  'Montgomery, Alabama',
+  'Juneau, Alaska',
+  'Phoenix, Arizona',
+  'Little Rock, Arkansas',
+  'Sacramento, California',
+  'Denver, Colorado',
+  'Hartford, Connecticut',
+  'Dover, Delaware',
+  'Tallahassee, Florida',
+  'Atlanta, Georgia',
+  'Honolulu, Hawaii',
+  'Boise, Idaho',
+  'Springfield, Illinois',
+  'Indianapolis, Indiana',
+  'Des Moines, Iowa',
+  'Topeka, Kansas',
+  'Frankfort, Kentucky',
+  'Baton Rouge, Louisiana',
+  'Augusta, Maine',
+  'Annapolis, Maryland',
+  'Boston, Massachusetts',
+  'Lansing, Michigan',
+  'Saint Paul, Minnesota',
+  'Jackson, Mississippi',
+  'Jefferson City, Missouri',
+  'Helena, Montana',
+  'Lincoln, Nebraska',
+  'Carson City, Nevada',
+  'Concord, New Hampshire',
+  'Trenton, New Jersey',
+  'Santa Fe, New Mexico',
+  'Albany, New York',
+  'Raleigh, North Carolina',
+  'Bismarck, North Dakota',
+  'Columbus, Ohio',
+  'Oklahoma City, Oklahoma',
+  'Salem, Oregon',
+  'Harrisburg, Pennsylvania',
+  'Providence, Rhode Island',
+  'Columbia, South Carolina',
+  'Pierre, South Dakota',
+  'Nashville, Tennessee',
+  'Austin, Texas',
+  'Salt Lake City, Utah',
+  'Montpelier, Vermont',
+  'Richmond, Virginia',
+  'Olympia, Washington',
+  'Charleston, West Virginia',
+  'Madison, Wisconsin',
+  'Cheyenne, Wyoming',
+];
+
+examples.get('/autosuggest/search_list', upload.none(), handleSearchRequest);
```

Comment:
For sample specific APIs, you can add an express router directly in the sample folder (see amp-live-list sample, but without SampleRenderer).